README.md

```diff
@@ -1 +1,129 @@
-# book-stores
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[React-Book-store\] ](#-react-book-store-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Installation and Setup](#installation-and-setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [📝 License ](#-license-)
+
+# 📖 [React-Book-store] <a name="about-project"></a>
+
+
+Book-Stores The JavaScript project is a bookstore app.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Book Stores]**
+- **[Quote]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
+
+Download the ZIP from this location, or run the following git command to clone the files to your machine:
+
+git clone:
+```sh
+  https://github.com/obibaadoma/book-stores
+```
+
+### Installation and Setup
+
+install the dependencies with:
+```sh
+  cd math-magician
+  npm install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm run test
+```
+
+
+### Deployment
+
+You can deploy this project using:
+
+```sh
+ npm run build
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+👤 **Author**
+
+- GitHub: (https://github.com/obibaadoma)
+- Twitter: (https://twitter.com/obibaadoma)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/derek-akrasi-konadu-187453151/)
+
+ Future Features
+Describe 1 - 3 features you will add to the project.
+
+ [Will include styling]
+
+(back to top)
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+I would like to thank...
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you add **Acknowledgement**, **Show your support** and **Contributing** sections to your readme file to make your readme descriptive and professional 👍